WordPress/Classes/ViewRelated/NUX/UnifiedProloguePages.swift

```diff
@@ -80,13 +81,15 @@ class UnifiedProloguePageViewController: UIViewController {
             return
         }
 
+        configureTitleFont()
+
         if traitCollection.horizontalSizeClass == .compact {
 
-            NSLayoutConstraint.deactivate([contentViewHeightConstraint ?? NSLayoutConstraint()])
+            NSLayoutConstraint.deactivate([contentViewHeightConstraint ?? NSLayoutConstraint(), titleWidthConstraint ?? NSLayoutConstraint()])
```

Comment:
I just tested and it looks as though we'll crash if we try and activate empty constraints:

```
2021-04-02 20:31:02.485889+0100 WordPress[5281:17160032] *** Terminating app due to uncaught exception 'NSGenericException', reason: 'Unable to activate constraint with anchors (null) and (null) because they have no common ancestor.  Does the constraint or its anchors reference items in different view hierarchies?  That's illegal.'
```

Could we make these non-optional? Or implicitly unwrapped?